### PR TITLE
Add possibility to enable Persistent Local Storage using Ansible

### PR DESCRIPTION
**Pull request by dabelenda**

### CODE REVIEW

**Comment by dabelenda** [comment]
ping @openshift/storage (don't know if it works with private teams)

File: playbooks/openshift-hosted/private/create_persistent_volumes.yml [hunk 1]
```diff
@@ -3,3 +3,26 @@
...
+  roles:
+  - role: openshift_persistentlocalstorage
+    when: openshift_persistentlocalstorage_enabled | default(False) | bool
+
+- name: Create Hosted Resources - Persistent Local Storage Classes
+  hosts: nodes
+  tasks:
+  - name: Create Persistent Local Storage Classes Directories
+    file:
+      path: "/mnt/local-storage/{{ item }}"
```
Comment (vrutkovs) [inline playbooks/openshift-hosted/private/create_persistent_volumes.yml]: path should be configurable

**Comment by vrutkovs** [comment]
bot, test pull request

**Comment by vrutkovs** [comment]
@rh-atomic-bot bot, test pull request

File: roles/openshift_persistentlocalstorage/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                      apiVersion: v1
+                      fieldPath: spec.nodeName
+                - name: MY_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      apiVersion: v1
+                      fieldPath: metadata.namespace
+                - name: VOLUME_CONFIG_NAME
+                  value: local-volume-provisioner-config
+                image: quay.io/external_storage/local-volume-provisioner:v1.0.1
```
Comment (vrutkovs) [inline roles/openshift_persistentlocalstorage/tasks/main.yml]: Image location and version should be parametrized

File: inventory/hosts.example [hunk 3]
```diff
@@ -585,6 +585,11 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # openshift_storageclass_name=gp2
 # openshift_storageclass_parameters={'type': 'gp2', 'encrypted': 'false'}
 #
+# PersistentLocalStorage
```
Comment (vrutkovs) [inline inventory/hosts.example]: example hosts should include `openshift_persistentlocalstorage_path` (with default value)

**Comment by dabelenda** [comment]
@vrutkovs   I guess the duplication of the default value comes from my inexperience with ansible, but I do not see how to reduce it further than what I already did (I removed the default value from the role. I thought that roles should be independent from the playbook and thus put the defaults there, too)

**Pull request merged**